e2e-tests/desktop/tests/credentials.spec.js

```diff
@@ -303,11 +303,11 @@ test.describe('Credential Panel tests', () => {
     // Empty string and null should not be visible
     await expect(
       authedPage.getByRole('listitem').filter({ hasText: 'nested.key3' }),
-    ).not.toBeVisible();
+    ).toBeHidden();
```

Comment:
I believe this is from a recent PR that was merged into main

Reply:
You are right. I merged main into this branch to update, and now is showing this as change I will look into it, it shouldn't show it.